src/sql/workbench/contrib/backup/browser/backup.component.ts

```diff
@@ -740,7 +740,8 @@ export class BackupComponent extends AngularDisposable {
 	}
 
 	private setDefaultBackupName(): void {
-		if (this.backupNameBox && (!this.backupNameBox.value || this.backupNameBox.value.trim().length === 0)) {
+		const suggestedNamePrefix = this.databaseName + '-' + this.getSelectedBackupType();
```

Comment:
when the backup type is "Transaction Log" should the space be replaced with a hyphen?

Reply:
We can - but it scripts correctly and is allowed with space too.